src/transient_groundwater.cpp

```diff
@@ -476,14 +473,15 @@ static PetscErrorCode FormJacobianLocal(DMDALocalInfo* info, PetscScalar** x, Ma
   //    locally (but any non-local elements will be sent to the
   //    appropriate processor during matrix assembly).
   //  - Here, we set all entries for a particular row at once.
-  PetscCall(DMDAVecGetArray(da, user->mask, &my_mask));
+  PETSC_CHECK(DMDAVecGetArray(user->da, user->mask, &my_mask));
 
   for (auto j = info->ys; j < info->ys + info->ym; j++) {
     for (auto i = info->xs; i < info->xs + info->xm; i++) {
       MatStencil row{.k = 0, .j = j, .i = i, .c = 0};
 
       // boundary points
       if (my_mask[j][i] == 0) {
+        PetscScalar local_v[9];
```

Comment:
It makes me nervous that this might not be getting initialized correctly in the old or the new code.

Reply:
@KCallaghan